engines/kinduction.cpp

```diff
@@ -109,14 +112,14 @@ bool KInduction::inductive_step(int i)
 
 Term KInduction::simple_path_constraint(int i, int j)
 {
-  assert(ts_->statevars().size());
+  assert(ts_.statevars().size());
 
   Term disj = false_;
-  for (auto v : ts_->statevars()) {
-    Term vi = unroller_.at_time(v, i);
-    Term vj = unroller_.at_time(v, j);
-    Term eq = solver_->make_term(PrimOp::Equal, vi, vj);
-    Term neq = solver_->make_term(PrimOp::Not, eq);
+  for (const auto &v : ts_.statevars()) {
+    const Term &vi = unroller_.at_time(v, i);
+    const Term &vj = unroller_.at_time(v, j);
+    const Term &eq = solver_->make_term(PrimOp::Equal, vi, vj);
```

Comment:
Can this be a `const` reference to a term that was just created? It seems like the object would be destroyed.